.gitignore

```diff
@@ -1,5 +1,11 @@
+.idea/
 .gradle
 /build/
+/*/build
+/*/out
+/*/*.iml
+/*.iml
```

Comment:
Something like **/*.iml should work for all .iml files in all directories. If I remember it right.

Reply:
Thanks!